extension/popup/popup.js

```diff
@@ -1,17 +1,30 @@
 import { createBookmark } from './services/BookmarkService.js';
-import { getActiveGroupInfo, getTabsForCurrentWindow } from './services/TabService.js';
+import { getActiveGroupInfo, getTabsForCurrentWindow, generateBookmarkName } from './services/TabService.js';
+
+const getBookmarkTitle = async (payload) => {
+  return (document.getElementById("inputTitle").value)? document.getElementById("inputTitle").value: await generateBookmarkName(payload.tabs.length);   
+}
+
+const awake = async () => {
+  const groupInfo = await getActiveGroupInfo();
+  if (groupInfo && groupInfo.title) {
+    document.getElementById("inputTitle").value = groupInfo.title
```

Comment:
Prefilling the field like this is kinda annoying imo, how about a placeholder and setting our fallback on creation in case the trimmed custom title is still an empty string

Reply:
I would expect, that the title should be ok in most of the cases, but we could also display it only as a hint